M2/Macaulay2/m2/last.m2

```diff
@@ -63,6 +63,7 @@ addStartFunction( () -> (
 	       	    hd := "with packages: ";
 	       	    stderr << hd << wrap(printWidth-#hd, concatenate between_", " sort apply(relevant,toString)) << endl;
```

Comment:
Replace with
```
stderr << "(Type `loadedPackages` for the list of preloaded packages.)" << endl;
```

Reply:
I'm not fully convinced one way or another about printing the loaded packages, but if we don't print them, I don't think I would want a parenthesized message in its place either.